### PR TITLE
Add a project for testing utilities

### PR DESCRIPTION
**Pull request by tomrijnbeek**
This PR adds a new project that can contain testing utilities. These testing utilities are meant to be used to make testing easier if you use some of our types (such as Maybe) by abstracting away common checks.

### CODE REVIEW

File: testing/Class1.cs [hunk 1]
```diff
@@ -0,0 +1,4 @@
+﻿namespace Bearded.Utilities.Testing
+{
+    public class Class1 { }
```
Comment (amulware) [inline testing/Class1.cs]: New phone, who dis?

Comment (tomrijnbeek) [reply testing/Class1.cs]: I wasn't sure if the project would even build without any classes, and I wanted to keep the project creation PR separate from the implementation of the first helper /shrug

**Comment by tomrijnbeek** [comment]
Now ready for review as new artifact is correctly picked up.

File: testing/Class1.cs (hunk 1 above)
Comment (tomrijnbeek) [reply testing/Class1.cs]: Or maybe the builder doesn't care...

**Pull request merged**